src/Microsoft.Azure.WebJobs.Host/Singleton/SingletonManager.cs

```diff
@@ -103,8 +104,6 @@ internal TimeSpan MinimumLeaseRenewalInterval
 
         public async virtual Task<object> TryLockAsync(string lockId, string functionInstanceId, SingletonAttribute attribute, CancellationToken cancellationToken, bool retry = true)
         {
-            _trace.Verbose(string.Format(CultureInfo.InvariantCulture, "Waiting for Singleton lock ({0})", lockId), source: TraceSource.Execution);
```

Comment:
What do you guys (@fabiocav @MikeStall) think about defining a set of ExtendedTraceLevel constants so that we can have a higher Diagnostic level (level 5). While not part of the System.Diagnostic.TraceLevel enum, we can still assign such an int. That would allow us to add TraceWriter.Diagnostic(). We could then retain these log statements and switch them on if needed.